docs/view.md

```diff
@@ -211,3 +212,38 @@ it produces:
 ```html
 <my-greeting><h1>Hello World</h1></my-greeting>
 ```
+
+### Omitting view entirely
+
+If the user does not provide a view property,
+the content -whether defined or passed inline, will be rendered instead.
+
+This means a component like this:
+
+```js
+Component.extend({
+	tag: "my-greeting",
```

Comment:
extra tab here?

Reply:
Not sure why this is showing up in Github, it doesn't show on my local text editor